cmd/gaia/app/sim_test.go

```diff
@@ -238,27 +236,20 @@ func appStateRandomizedFn(r *rand.Rand, accs []simulation.Account, genesisTimest
 	stakingGenesis.Pool.NotBondedTokens = sdk.NewInt((amount * numAccs) + (numInitiallyBonded * amount))
 	stakingGenesis.Validators = validators
 	stakingGenesis.Delegations = delegations
+	modulesGenesis[staking.ModuleName] = cdc.MustMarshalJSON(stakingGenesis)
 
+	// TODO make use NewGenesisState
```

Comment:
is this a reminder for this PR ?

Reply:
No this is a reminder for the codebase (as a part of existing issues) - this requires a more thorough understanding as there are some missing fields